src/modules/feedback/components/generalFeedback/GeneralFeedbackPanel.js

```diff
@@ -0,0 +1,176 @@
+
+		const translate = (key) => this._translationService.translate(key);
+
+		const onRatingChange = (event) => {
+			const {
+				detail: { rating },
+				target: { parentNode }
+			} = event;
+
+			this._addVisitedClass(parentNode);
+			this.signal(Update_Rating, this._securityService.sanitizeHtml(rating));
+		};
+
+		const handleEmailChange = (event) => {
+			const { value, parentNode } = event.target;
+			this._addVisitedClass(parentNode);
+
+			this.signal(Update_EMail, this._securityService.sanitizeHtml(value));
+		};
+
+		const onDescriptionChange = (event) => {
+			const { value, parentNode } = event.target;
+			this._addVisitedClass(parentNode);
+
+			this.signal(Update_Description, this._securityService.sanitizeHtml(value));
+		};
+
+		const onSubmit = () => {
+			this.shadowRoot.querySelectorAll('.ba-form-element').forEach((el) => el.classList.add(BA_FORM_ELEMENT_VISITED_CLASS));
+
+			const descriptionElement = this.shadowRoot.getElementById('description');
+			const emailElement = this.shadowRoot.getElementById('email');
+
+			if (descriptionElement.reportValidity() && emailElement.reportValidity()) {
+				this._saveGeneralFeedback(new GeneralFeedback(generalFeedback.description, generalFeedback.email, generalFeedback.rating));
+			}
+		};
+
+		return html`
+			<style>
+				${css}
+			</style>
+
+			<h2 id="feedbackPanelTitle">${translate('feedback_generalFeedback')}</h2>
+
+			<div class="ba-form-element">
+				<label for="rating" class="control-label">${translate('feedback_generalFeedback_rating')}</label>
+				<ba-stars-rating-panel
+					id="rating"
+					@change="${onRatingChange}"
+					placeholder="${translate('feedback_generalFeedback_rating')}"
+					required
+				></ba-stars-rating-panel>
+			</div>
+
+			<div class="ba-form-element">
+				<label for="description" class="control-label">${translate('feedback_changeDescription')}</label>
+				<textarea
+					type="text"
+					id="description"
+					placeholder="${translate('feedback_changeDescription')}"
+					.value="${generalFeedback.description}"
+					@input="${onDescriptionChange}"
+					required
+				></textarea>
+				<i class="bar"></i>
+				<i class="icon error"></i>
+				<label class="helper-label">${translate('feedback_required_field_helper')}</label>
+				<label class="error-label">${translate('feedback_required_field_error')}</label>
+			</div>
+
+			<div class="ba-form-element">
+				<input type="email" id="email" placeholder="${translate('feedback_eMail')}" .value="${generalFeedback.email}" @input="${handleEmailChange}" />
+				<label for="email" class="control-label">${translate('feedback_eMail')}</label>
+				<i class="bar"></i>
+				<i class="icon error"></i>
+				<label class="helper-label">${translate('feedback_eMail_helper')}</label>
+				<label class="error-label">${translate('feedback_eMail_error')}</label>
+			</div>
+
+			<p id="feedback_mapFeedback_disclaimer" class="map-feedback__disclaimer" id="mapFeedback_disclaimer">
```

Comment:
there should be only one id...

In GeneralFeedbackPanel.js should 'mapFeedback' not occur.

If the name is an Element-ID -> rename id to `generalFeedback*`.
If the name is an i18n-ID -> rename the id in ALL places : `feedback_mapFeedback_disclaimer` -> `feedback_disclaimer`